<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bdMRMainWindow</class>
 <widget class="QMainWindow" name="bdMRMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Rename</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="mainGrp">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="masksGrp">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>150</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>301</width>
              <height>103</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="1">
              <widget class="QLineEdit" name="inputPrefix"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Prefix:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="inputRenameMask"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Rename Mask:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Sufix:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Count Starts at:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="inputSufix"/>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="inputStartCounter"/>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="renameBtn">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>120</y>
              <width>301</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Rename</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="searchReplaceGrp">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>301</width>
              <height>100</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Search:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="inputSearch"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Replace:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="inputReplace"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Prefix:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Sufix:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="inputPrefixReplace"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="inputSufixReplace"/>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="replaceBtn">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>120</y>
              <width>301</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Replace and/or add sufix/prefix</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="previewGrp">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>280</height>
            </size>
           </property>
           <property name="title">
            <string>Preview</string>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>301</width>
              <height>244</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="text">
                <string>Original</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_8">
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="text">
                <string>Renamed</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QListWidget" name="listOriginal"/>
             </item>
             <item row="1" column="1">
              <widget class="QListWidget" name="listPreview"/>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QPushButton" name="updateBtn">
               <property name="text">
                <string>Update Selected</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>inputRenameMask</tabstop>
  <tabstop>inputPrefix</tabstop>
  <tabstop>renameBtn</tabstop>
  <tabstop>inputSearch</tabstop>
  <tabstop>replaceBtn</tabstop>
  <tabstop>listOriginal</tabstop>
  <tabstop>listPreview</tabstop>
  <tabstop>updateBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
